content/en/about/contributing/style/_index.md

```diff
@@ -22,22 +22,28 @@ if any:
 
 ## Markdown writing guidelines
 
-Documentation pages have to be written in markdown compliant with
+Documentation pages have to be written in markdown, compliant with
 [CommonMark](https://spec.commonmark.org/0.29/) and
 [GitHub Flavored Markdown](https://github.github.com/gfm/).
 
 ### Basic rules
 
-- Headings start at level 2 (`##`), as level 1 (`#`) is the title of the page
-- Do not capitalize each word in heading titles, unless it is a service name
+- Headings must start at level 2 (`##`), as level 1 (`#`) is the title of the page
 - Lines should be wrapped at 80 characters
-- Sentences should be separated by one space only
-- Indent is made with tabs not with spaces
-- Lists should be using `-` not `*`
+- Sentences must be separated by one space only
+- Indent is made with tabs, not with spaces
```

Comment:
Wow, I think I overlooked this one, we definitely do not want to mix spaces and tabs, especially as tab width can vary, so it's the other way around: indent should be made with spaces, not spaces

Reply:
And and indent level is typically two spaces

Reply:
@gwarf so spaces, not tabs, right?